CONTRIBUTING.md

```diff
@@ -0,0 +1,166 @@
+
+## PR Process
+
+Every PR should be annotated with an icon indicating whether it's a:
+
+-   Breaking change: :warning: (`:warning:`)
+-   Non-breaking feature: :sparkles: (`:sparkles:`)
+-   Patch fix: :bug: (`:bug:`)
+-   Documentation changes (user or developer): :book: (`:book:`)
+-   Infra/Tests/Other: :seedling: (`:seedling:`)
+-   No release note: :ghost: (`:ghost:`)
+
+Use :ghost: (no release note) only for the PRs that change or revert unreleased
+changes, which don't deserve a release note. Please don't abuse it.
+
+Prefer using the `:xyz:` aliases over the equivalent emoji directly when possible.
+
+Individual commits should not be tagged separately, but will generally be
+assumed to match the PR. For instance, if you have a bugfix in with a breaking
+change, it's generally encouraged to submit the bugfix separately, but if you must put them in one PR, you should mark the whole PR as breaking.
+
+> [!NOTE]
+> Once a maintainer reviews your code, please address feedback without rebasing when possible.
+> This includes [synchronizing your PR](https://docs.github.com/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/keeping-your-pull-request-in-sync-with-the-base-branch)
+> with `main`. The GitHub review experience is much nicer with traditional merge commits.
+
+## Changelog and versioning
+
+**This work is done by the maintainers exclusively.**
+
+In order to generate a new release, it is recommended to use the commands:
+
+
+```
+npm run release:minor
+npm run release:patch
+npm run release:major
+```
+
+
+This includes all the changes in the [CHANGELOG](./CHANGELOG.md) and ensures that the `package.json` and `package-lock.json` are up to date.
+
+You can discard the tag that has been generated locally, as we won't use it.
+
+## Releasing a new version
+
+**This work is done by the maintainers exclusively.**
+
+It is important to ensure that the `package.json`, `package-lock.json` and `CHANGELOG.md` are correct and include all the details for the new release in the `main` branch.
+
+In order to create a new release, follow these steps:
+
+1. Use the GitHub web UI for [new releases](https://github.com/ossf/scorecard-monitor/releases/new).
+2. Mark `Publish this Action to the GitHub Marketplace` as we want to deliver this to our users.
+3. Target the new release version, like `v.1.0.3-beta5`. Note that you can use metadata like `-beta5` and you must include `v` as prefix.
+4. Mark `Set as the latest release`
+5. (Optionally) mark `Set as a pre-release` if it is non-production ready.
+6. :bulb: **Hint:** You can check another release ([example](https://github.com/ossf/scorecard-monitor/releases/tag/v2.0.0-beta7)) to follow the style for title and description (_Main Changes, PRs, New contributions_). 
+7. Click the `Generate release notes` button in the top right to automatically populate the release description.
```

Comment:
Let's consider moving this into a separate document e.g., `docs/releasing.md`

Reply:
Done! Moved to `RELEASE.md` following the scorecard repo conventions in https://github.com/ossf/scorecard-monitor/pull/85/commits/1aa027b9123b416dbee65ae95c1ffae692310d39